CHANGELOG.md

```diff
@@ -96,6 +96,7 @@ https://hyperledger.jfrog.io/artifactory/besu-binaries/besu/23.7.3/besu-23.7.3.z
 - JSON output is now compact by default. This can be overridden by the new `--json-pretty-print-enabled` CLI option. [#5766](https://github.com/hyperledger/besu/pull/5766)
 - New `eth_getBlockReceipts` JSON-RPC method to retrieve all transaction receipts for a block in a single call [#5771](https://github.com/hyperledger/besu/pull/5771) 
 - Add new methods to `OperationTracer` to capture contexts enter/exit [#5756](https://github.com/hyperledger/besu/pull/5756)
+- Ethereum Classic Spiral network upgrade [#6078](https://github.com/hyperledger/besu/pull/6078)
```

Comment:
this should be in 23.10.2

Reply:
Sure, I'll move it, but do you think it would be possible to include this into `23.10.1`?

Reply:
sorry, that was already cut last week and burn-in just about finished. Ask in the besu-release channel re timing for the next release https://discord.com/channels/905194001349627914/943252457063075880/1164641743782805525

Reply:
@macfarla I added a new section for `23.10.2` and moved the entry there